ash_token/.gitignore

```diff
@@ -0,0 +1,17 @@
+# Compiler files
+cache/
+out/
+
+# Ignores development broadcast logs
+!/broadcast
+/broadcast/*/31337/
+/broadcast/**/dry-run/
+
+# Docs
+docs/
+
+# Dotenv file
+.env
+
+# libs
+lib/
```

Comment:
Instead of this, create and commit an empty `lib` directory so that it already exists when we use it in the tutorial.